libinstpatch/CMakeLists.txt

```diff
@@ -345,15 +345,13 @@ if ( MACOSX_FRAMEWORK )
 elseif( MINGW OR WIN32 )
   set_target_properties ( libinstpatch
     PROPERTIES
-      PREFIX ""
-      OUTPUT_NAME ${INSTPATCH_INSTALL_TARGET}
+      OUTPUT_NAME "instpatch-${LIB_VERSION_CURRENT}"
       VERSION ${LIB_VERSION_INFO}
   )
```

Comment:
Ok thanks, that looks better. One minor issue: The `libinstpatch-2.dll` is now called `instpatch-2.dll`. If possible I would like to avoid that change. Could you try to change this `elseif` to the following, pls:

```cmake
elseif( MINGW OR WIN32 )
  set_target_properties ( libinstpatch
    PROPERTIES
      ARCHIVE_OUTPUT_NAME "instpatch-${LIB_VERSION_CURRENT}"
      PREFIX "lib"
      OUTPUT_NAME "instpatch-${LIB_VERSION_CURRENT}"
      VERSION ${LIB_VERSION_INFO}
  )
```

This should generate an import lib `instpatch-2.lib` and a `libinstpatch-2.dll`. The import lib should be found by pkg-config. Similar as we do it in fluidsynth.

Reply:
Pushed.